projects/plugins/jetpack/extensions/blocks/payment-buttons/save.js

```diff
@@ -1,6 +1,11 @@
 import { useBlockProps, useInnerBlocksProps } from '@wordpress/block-editor';
+import classNames from 'classnames';
 
-export default function save() {
-	const innerBlocksProps = useInnerBlocksProps.save( useBlockProps.save() );
+export default function save( { attributes } ) {
+	const className = classNames( {
+		'has-custom-font-size': !! attributes?.fontSize || attributes?.style?.typography?.fontSize,
```

Comment:
Are you planning on re-using some logic from the `jetpack/button` block to determine the class names? If so, maybe we could extract the logic from the `jetpack/button` block to a shared component so that both blocks can use the same logic. WDYT?

Reply:
Not right now. I wanted to see if this approach was okayed first, then I think an alignment could be made as a follow-up, so long as there's no migration issues. i.e. use has-custom-font-size and a css rule.